.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: scala
+
+jdk: oraclejdk8
+
+matrix:
+  include:
+    - name: "sbt-restli [sbt 0.13]"
+      env: SBT_COMMAND="++2.10.7 ^^0.13.17 sbtRestli/scripted"
+    - name: "sbt-restli [sbt 1.0]"
```

Comment:
```suggestion
    - name: "sbt-restli [sbt 1.x]"
```

Reply:
Based on how official sbt documentation categorizes the version, 1.x would be more accurate.